src/shell/store/user.js

```diff
@@ -44,17 +40,7 @@ export function user(
     //DRY Creating latest_edits reducer for dashboard
     case "FETCH_USER_LOGS_SUCCESS":
```

Comment:
I think this whole case can be removed as it was added with the dashboard work. https://github.com/zesty-io/manager-ui/commit/14b29a0754cf2b1575f58f0dbd957bbe544ec169#diff-da848ab72737014691325cb22eb36812b100596c01de290443fa2b66f52177d4R45

Reply:
Okay removed, I though it was necessary for logs store but I now see log has it's own action call